.github/workflows/ci.yaml

```diff
@@ -28,7 +28,7 @@ jobs:
           mvn install -B -ntp -DskipTests -Dclirr.skip -Dcheckstyle.skip
       - name: Integration Tests
         run: |
-          bazelisk --batch test //test/integration/...
+          bazelisk --batch test //test/integration/... 
```

Comment:
Remove unnecessary space ad the end. Same for Line 126.

Reply:
`git diff origin/main` is your friend.

Reply:
All done.